appinfo/routes/routesRoomController.php

```diff
@@ -91,6 +91,8 @@
 		['name' => 'Room#verifyDialInPin', 'url' => '/api/{apiVersion}/room/{token}/verify-dialin', 'verb' => 'POST', 'requirements' => $requirementsWithToken],
 		/** @see \OCA\Talk\Controller\RoomController::verifyDialOutNumber() */
 		['name' => 'Room#verifyDialOutNumber', 'url' => '/api/{apiVersion}/room/{token}/verify-dialout', 'verb' => 'POST', 'requirements' => $requirementsWithToken],
+		/** @see \OCA\Talk\Controller\RoomController::directDialIn() */
+		['name' => 'Room#directDialIn', 'url' => '/api/{apiVersion}/room/{token}/direct-dial-in', 'verb' => 'POST', 'requirements' => $requirementsWithToken],
```

Comment:
Which `token` should I send here? The room does not exist at the time of the request.

Reply:
that's a correct oversight. Should be a URL without a token